package.json

```diff
@@ -82,13 +73,8 @@
     "scroll-to-element": "^2.0.0",
     "sentry-dreamy-components": "^1.0.0",
     "sprintf-js": "1.0.3",
-    "style-loader": "0.12.4",
-    "svg-sprite-loader": "^3.4.1",
     "svgo": "^1.0.3",
-    "svgo-loader": "^2.1.0",
     "u2f-api": "0.2.3",
-    "webpack": "^3.5.6",
```

Comment:
No. This is a little confusing actually. Since sentry isn't a node application, what package.json is *really* for is defining the libraries needed to "build" sentry's frontend, so `node_modules` aren't being shipped, but code is being pulled out of them and packaged into our final `app.js`.